msibi/pair.py

```diff
@@ -23,113 +20,72 @@ class Pair(object):
     type1 : str, required
         The name of one particle type on the particle pair.
         Must match the names found in the State's .gsd trajectory file.
-        See  gsd.hoomd.ParticleData.types 
+        See  gsd.hoomd.ParticleData.types
     type2 : str, required
         The name of one particle type on the particle pair.
         Must match the names found in the State's .gsd trajectory file.
-        See  gsd.hoomd.ParticleData.types 
-    potential : 
+        See  gsd.hoomd.ParticleData.types
+    potential :
 
 
     Attributes
     ----------
     name : str
         Pair name.
-    states : dict
-        Dictionary of each state added for this pair using
-        the Pair.add_state() method
     potential : func
         Values of the potential at every pot_r.
-
     """
 
     def __init__(self, type1, type2, potential, head_correction_form="linear"):
         self.type1 = str(type1)
         self.type2 = str(type2)
         self.name = f"{self.type1}-{self.type2}"
         self.potential_file = ""
-        self.states = dict()
-        if isinstance(potential, string_types):
+        self._states = dict()
+        if isinstance(potential, str):
             self.potential = np.loadtxt(potential)[:, 1]
             # TODO: this could be dangerous
         else:
             self.potential = potential
         self.previous_potential = None
         self.head_correction_form = head_correction_form
 
-    def add_state(
+    def _add_state(
         self,
```

Comment:
Can we put this on a single line? It would be no where close to 80 columns.

Reply:
heck ya